### ISSUE TITLE
persist plugin typescript path

### ISSUE THREAD

**Issue by matt-erhart** [rematch/rematch#518]
index.d.ts should be
```ts
import { Plugin } from '@rematch/core';
export declare const getPersistor: () => any;
declare const reactNavigationPlugin: (
  config?: {}
) => Plugin<
  import('../../core/src/typings').Models,
  import('../../core/src/typings').Action<any, any>
>;
export default reactNavigationPlugin;
```
but is this:
```ts
import { Plugin } from '@rematch/core';
export declare const getPersistor: () => any;
declare const reactNavigationPlugin: (
  config?: {}
) => Plugin<
  import('../node_modules/@rematch/core/src/typings').Models,
  import('../node_modules/@rematch/core/src/typings').Action<any, any>
>;
export default reactNavigationPlugin;
```

**Comment by ShMcK** [comment]
Must be a product of the build process, will fix.

**Comment by RunningCoderLee** [comment]
I forked this repo and run build command in `plugins/persist` is ok 

the `index.d.ts` in `dist` is correct as below:

```ts
import { Plugin } from '@rematch/core';
export declare const getPersistor: () => any;
declare const persistPlugin: (config: {}, pesistStoreConfig: any, callback: any) => Plugin<import("@rematch/core").Models, import("@rematch/core").Action<any, any>>;
export default persistPlugin;
```
but `index.d.ts` file in the newest version (1.1.0) is this: 

```ts
import { Plugin } from '@rematch/core';
export declare const getPersistor: () => any;
declare const reactNavigationPlugin: (config?: {}) => Plugin<import("../node_modules/@rematch/core/src/typings").Models, import("../node_modules/@rematch/core/src/typings").Action<any, any>>;
export default reactNavigationPlugin;
```

does this issue have any update？

**Comment by ShMcK** [comment]
I'll clean this up and release @rematch/persist@1.1.1

**Comment by RunningCoderLee** [comment]
Many thanks your update but it doesn't resolve this issue completely

the `types` field in `package.json` change to `src/typings.d.ts` but the declaration file that uploaded to npm is `src/index.d.ts`

**Comment by ShMcK** [comment]
Sorry, I've fixed the path in package.json and re-released.

Let me know if this resolves the issue in @rematch/persist @1.1.2
